scripts/dotnet-cli-build/PublishTargets.cs

```diff
@@ -101,6 +101,7 @@ public static BuildTargetResult PublishCoreHostPackages(BuildTargetContext c)
             {
                 var hostBlob = $"{Channel}/Binaries/{CliNuGetVersion}/{Path.GetFileName(file)}";
                 AzurePublisherTool.PublishFile(hostBlob, file);
+                Console.WriteLine($"Publishing package {hostBlob} to Azure.");
```

Comment:
I see, I've been trying to figure out what the build scripts use to write to console.